src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -10777,7 +10777,7 @@ namespace HVACUnitarySystem {
 				// zone equipment needs to set flow since no other device regulates flow (ZoneHVAC /= AirLoopEquipment)
 				if ( !UnitarySystem( UnitarySysNum ).AirLoopEquipment ) {
 					Node( InletNode ).MassFlowRate = AverageUnitMassFlow;
-					Node( InletNode ).MassFlowRateMaxAvail = AverageUnitMassFlow;
+//					Node( InletNode ).MassFlowRateMaxAvail = AverageUnitMassFlow; #5531 max iterations
```

Comment:
Do we have a test file with UnitarySystem as zone equipment?  This line may need to stay.